cloudify_cli/config/parser_config.py

```diff
@@ -19,52 +19,95 @@
 
 from cloudify_cli import commands as cfy
 from cloudify_cli.config import completion_utils
-from cloudify_cli.config import argument_utils
+from cloudify_cli.config.argument_utils import remove_type
+from cloudify_cli.config.argument_utils import remove_completer
+from cloudify_cli.config.argument_utils import make_required
+from cloudify_cli.config.argument_utils import make_optional
 from cloudify_cli.constants import DEFAULT_REST_PORT
+from cloudify_cli.constants import DEFAULT_BLUEPRINT_FILE_NAME
+from cloudify_cli.constants import DEFAULT_BLUEPRINT_PATH
+from cloudify_cli.constants import DEFAULT_INPUTS_PATH_FOR_INSTALL_COMMAND
+from cloudify_cli.constants import DEFAULT_TIMEOUT
+from cloudify_cli.constants import DEFAULT_PARAMETERS
+from cloudify_cli.constants import DEFAULT_TASK_THREAD_POOL_SIZE
+from cloudify_cli.constants import DEFAULT_INSTALL_WORKFLOW
+from cloudify_cli.constants import DEFAULT_UNINSTALL_WORKFLOW
 
 FORMAT_INPUT_AS_YAML_OR_DICT = 'formatted as YAML or as "key1=value1;key2=value2"'
 
 
+def manager_blueprint_path_argument():
+
+    # These are specific attributes to the manager blueprint path argument
+    argument = {
+        'metavar': 'BLUEPRINT_FILE',
+        'type': argparse.FileType(),
+        'completer': completion_utils.yaml_files_completer
+    }
+    hlp = "Path to the application's blueprint file." \
+          "Default: `{0}`".format(DEFAULT_BLUEPRINT_PATH)
+
+    # Update the specific 'manager blueprint path argument' attributes with
+    # those that are shared with the 'local blueprint path argument'
+    argument.update(local_blueprint_path_argument(hlp))
+
+    return argument
+
+
+def local_blueprint_path_argument(hlp):
+    return {
+        'dest': 'blueprint_path',
+        'required': True,
+        'help': hlp
+    }
+
+
 def blueprint_id_argument():
     return {
-        'metavar': 'BLUEPRINT_ID',
-        'type': str,
-        'help': 'The id of the blueprint',
+        'help': "The blueprint's id",
```

Comment:
This is not pettiness, this is very important.
But as we discussed, if there's a single quote inside the string, I'll use double quotes to delimiter it.